tests/integration/actions.go

```diff
@@ -367,6 +367,7 @@ func (tr TestRun) submitParamChangeProposal(
 		`--chain-id`, string(tr.chainConfigs[action.chain].chainId),
 		`--home`, tr.getValidatorHome(action.chain, action.from),
 		`--node`, tr.getValidatorNode(action.chain, action.from),
+		`--gas`, "900000",
```

Comment:
Is gas `auto` possible? What was the reason for 900000?

Reply:
@MSalopek can answer for this tx specifically, but I've previously needed to set a gas amount above the default for some integration test actions. Some of the on chain computations in these tests use a lot of gas